packages/components/src/dashicon/README.md

```diff
@@ -13,3 +21,372 @@ const MyDashicon = () => (
 	</div>
 );
 ```
+
+## Props
+
+The component accepts the following props:
+
+### icon
+
+The name of the icon. See the list below for all possible values.
```

Comment:
Listing and showing a preview of the icons in this README is a nice addition, but I believe that instead we should just link to the official docs (which also list the icons divided by categories)

```suggestion
The icon's `id`. For a complete list of all possible values, please refer to the [official docs](https://developer.wordpress.org/resource/dashicons/).
```

Reply:
This seems legit to me.

Reply:
Awesome! Let me know once you've addressed all the feedback, and I'll have another look at this PR :)